CMakeLists.txt

```diff
@@ -556,6 +556,18 @@ add_executable(test-script-asset-cache ${TEST_SCRIPT_ASSET_CACHE_SOURCES} "${CMA
 set_property(TARGET test-script-asset-cache PROPERTY PDB_NAME "test-script-asset-cache${VCPKG_PDB_SUFFIX}")
 endif()
 
+# === Target: verify_messages ===
+set(PS_SCRIPT_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/verifyMessages.ps1")
+set(SEARCH_DIR "${CMAKE_CURRENT_SOURCE_DIR}") 
+set(MESSAGE_FILE "${CMAKE_CURRENT_SOURCE_DIR}/locales/messages.es.json")
```

Comment:
Do we need these 3 very generic variable names? I think it would be just as clear or clearer to just expand them inline.

(Note that I think `ARTIFACT_MESSAGES_FILE` is OK because it describes exactly what it is and is unlikely to ever conflict with anything else)